cljfmt/test/cljfmt/core_test.cljc

```diff
@@ -64,7 +65,55 @@
     (is (= (reformat-string "(t/defn foo [x]\n(+ x 1))")
            "(t/defn foo [x]\n  (+ x 1))"))
     (is (= (reformat-string "(t/defrecord Foo [x]\nCloseable\n(close [_]\n(prn x)))")
-           "(t/defrecord Foo [x]\n  Closeable\n  (close [_]\n    (prn x)))")))
+           "(t/defrecord Foo [x]\n  Closeable\n  (close [_]\n    (prn x)))"))
+    (is (= (-> ["(ns example"
+                "(:require [thing.core :as t]))"
+                ""
+                "(t/defn foo [x]"
+                "(+ x 1))"
+                ""
+                "(defn foo [x]"
+                "(+ x 1))"]
+               (->> (str/join "\n"))
```

Comment:
It might look a little better if you used `str`, e.g.

```clojure
(str "(ns example\n"
     "(:require [thing.core :as t]))\n\n"
     "(t/defn foo [x]\n"
     "(+ x 1))\n\n"
     "(defn foo [x]\n"
     "(+ x 1))")
```